app/cypress/e2e/toolbar.cy.ts

```diff
@@ -0,0 +1,60 @@
+describe("Toolbar tests", () => {
+
+    beforeEach(() => {
+        sessionStorage.clear();
+    });
+
+    it("Verify toolbar items", () => {
+        cy.visit('/');
+        cy.findByRole('button', { name: 'OK' }).click();
+        const toolbarItems = ['Trade-off Tool for Sampling (TOTS)', 'Settings', 'Basemap', 'Login', 'Contact Us'];
+        toolbarItems.map((item) => {
+            cy.get('[data-testid="tots-toolbar"]').contains(item).should('exist');
+        });
+    });
+
+    it("Verify Settings", () => {
+        cy.visit('/');
+        cy.findByRole('button', { name: 'OK' }).click();
+
+        cy.findByRole('button', { name: 'Settings' }).should('exist').click({ force: true });
+
+        //Dimension
+        cy.findByText('Dimension').should('exist');
+        cy.get('#dimension-3d').check({ force: true });
+        cy.validateSession('tots_display_mode', 'dimensions', '3d');
+        cy.get('#dimension-2d').check({ force: true });
+        cy.validateSession('tots_display_mode', 'dimensions', '2d');
+
+        //Shape
+        cy.get('#shape-polygons').check({ force: true });
+        cy.validateSession('tots_display_mode', 'geometryType', 'polygons');
+        cy.get('#shape-points').check({ force: true });
+        cy.validateSession('tots_display_mode', 'geometryType', 'points');
```

Comment:
```suggestion
        cy.get('#shape-polygons').check({ force: true });
        cy.validateSession('tots_display_mode', 'geometryType', 'polygons');
        cy.get('#shape-points').check({ force: true });
        cy.validateSession('tots_display_mode', 'geometryType', 'points');
```
Can you also add a visual test for verifying points vs polygons?